File: sandbox.client/front/ts/model/ClientInfo.ts
export class ClientInfo {
    public id : string;
    public fullName : string;
    public charm : string;
    public age : number;
    public totalScore : number;
    public maxScore : number;
    public minScore : number;


    public assign(o: ClientInfo): ClientInfo {
        this.id = o.id;
        this.fullName = o.fullName;
        this.charm = o.charm;
        this.age = o.age;
        this.totalScore = o.totalScore;
        this.maxScore = o.maxScore;
        this.minScore = o.minScore;

        return this;
    }
}

File: sandbox.controller/src/kz/greetgo/sandbox/controller/model/ClientInfo.java
package kz.greetgo.sandbox.controller.model;


public class ClientInfo {
  public String id;
  public String fullName;
  public String charm;
  public int age;
  public double totalScore;
  public double maxScore;
  public double minScore;

  @Override
  public String toString() {
    return "ClientInfo{" +
      "id='" + id + '\'' +
      ", fullName='" + fullName + '\'' +
      ", charm='" + charm + '\'' +
      ", age=" + age +
      ", totalScore=" + totalScore +
      ", maxScore=" + maxScore +
      ", minScore=" + minScore +
      '}';
  }
}
